CONTRIBUTING.md

```diff
@@ -1,14 +1,13 @@
 Maintainers ([@open-telemetry/php-maintainers](https://github.com/orgs/open-telemetry/teams/php-maintainers)):
 
-- [Bob Strecansky](https://github.com/bobstrecansky), Mailchimp
+- [Bob Strecansky](https://github.com/bobstrecansky), Intuit
 
 Find more about the maintainer role in [community repository](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer)
 
 Approvers ([@open-telemetry/php-approvers](https://github.com/orgs/open-telemetry/teams/php-approvers)):
 
-- [Levi Morrison](https://github.com/morrisonlevi), Datadog
-- [Austin Schoen](https://github.com/AustinSchoen), Mailchimp
-- [Beniamin Calota](https://github.com/beniamin), eMag
+- [Timo Michna](https://github.com/tidal/)
```

Comment:
I'm not really feeling comfortable being called an "approver".

However what's the reason we keep such information in the CONTRIBUTING doc?

Reply:
People often look at the CONTRIBUTOR.md doc to see quickly who the contributors for the repo are without having to look all through github.  It's a convention we are mirroring from other CNCF repos.